### ISSUE TITLE
Attributes break colorization during typing

### ISSUE THREAD

**Issue by forki** [ionide/ionide-fsgrammar#89]
![image](https://user-images.githubusercontent.com/57396/42813795-6cb14ade-89c2-11e8-86a7-c7ec7c5511f5.png)

the `]` is already inserted and when you type the coloring is broken. Please add `]` as stop token in atttribute regex

@MangelMaxime

**Comment by MangelMaxime** [comment]
But isn't `>]` being the correct stop token ?

After, yes I can probably apply the requested fix for the attribute regex only.

**Comment by forki** [comment]
yes `]` isn't correct stop token. but this is about not breaking colorization during typing

**Comment by MangelMaxime** [comment]
Ok, will fix this one.

There is others places where the colorization is broken when typing if I remember correctly will try to limit thus when reworking the regexes.
